packages/block-editor/src/components/block-list/dropping-insertion-point.native.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * External dependencies
+ */
+import Animated, {
+	useSharedValue,
+	useAnimatedStyle,
+	withTiming,
+	useAnimatedReaction,
+	runOnJS,
+} from 'react-native-reanimated';
+
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { store as blockEditorStore } from '../../store';
+import { useBlockListContext } from './block-list-context';
+import styles from './dropping-insertion-point.scss';
+
+/**
+ * Dropping zone indicator component.
+ *
+ * This component shows where a block can be dropped when it's being dragged.
+ *
+ * @param {Object}                                        props                        Component props.
+ * @param {Object}                                        props.scroll                 Scroll offset object.
+ * @param {import('react-native-reanimated').SharedValue} props.hasStartedDraggingOver Whether or not the block has started moving.
+ * @param {import('react-native-reanimated').SharedValue} props.targetBlockIndex       Current block target index.
+ *
+ * @return {JSX.Element} The component to be rendered.
+ */
+export default function DroppingInsertionPoint( {
```

Comment:
If this component is only used in `BlockDraggable`, I'm wondering if we should locate the file `dropping-insertion-point.native.js` into `packages/block-editor/src/components/block-draggable`, wdyt?

Reply:
It makes sense to move it so I went ahead and applied the change in 57e822ccc5a14f13f3be2d1e3af50f79ca575b73